docs/supported-libraries.md

```diff
@@ -74,7 +74,7 @@ These are the supported libraries and frameworks:
 | [Guava ListenableFuture](https://guava.dev/releases/snapshot/api/docs/com/google/common/util/concurrent/ListenableFuture.html)              | 10.0+                              | [opentelemetry-guava-10.0](../instrumentation/guava-10.0/library)                                                                                                                                                                                                                                                                                                                       | Context propagation                                                                                                                     |
 | [GWT](http://www.gwtproject.org/)                                                                                                           | 2.0+                               | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [RPC Server Spans]                                                                                                                      |
-| [Hibernate](https://github.com/hibernate/hibernate-orm)                                                                                     | 3.3+                               | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                                                                    |
+| [Hibernate](https://github.com/hibernate/hibernate-orm)                                                                                     | 3.3+ (not including 7.0+ yet)      | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                                                                    |
```

Comment:
this is a bit misleading, instrumentation mostly works, it is only the tests that don't compile/fail

Reply:
```suggestion
| [Hibernate](https://github.com/hibernate/hibernate-orm)                                                                                     | 3.3+                               | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                                                                    |
```